sdk/core/Azure.Core.TestFramework/src/Azure.Core.TestFramework.csproj

```diff
@@ -12,6 +12,9 @@
     <PackageReference Include="Microsoft.NET.Test.Sdk" />
     <PackageReference Include="Moq" />
     <Reference Include="System.Web" Condition="'$(TargetFramework)' == 'net462' OR '$(TargetFramework)' == 'net47'" />
+
+    <PackageReference Include="Microsoft.Azure.AutoRest.CSharp" VersionOverride="3.0.0-beta.20231107.1" PrivateAssets="All" />
```

Comment:
Let's leave comments that it would be deleted afterwards.

Reply:
Would it be possible to share a little more information about the problem this is solving?  
I like @pshao25's comment calling out that it is temporary - at what point will it be removed - i.e. what needs to happen to resolve the issue this is addressing?

Thanks!